e2e-contracts/test-contracts.sh

```diff
@@ -61,6 +67,18 @@ if [ "$token" == 1 ]; then
     test brlc-token BRLCToken.complex
     test brlc-token BRLCTokenBridgeable
     test brlc-token USJimToken
+    test brlc-token InfinitePointsToken 
+    test brlc-token LightningBitcoin 
+    test brlc-token USJimToken 
+fi
+
+if [ "$pix" == 1 ]; then
+    test brlc-pix-cashier PixCashier
+fi
+
+if [ "$yield" == 1 ]; then
+    test brlc-yield-streamer BalanceTracker
+    test brlc-yield-streamer YieldStreamer
 fi
```

Comment:
this is all thats needed to make those tests? How are arguments passed?

Reply:
To run the tests you ask ? That is all, if you want to execute one contract in particular you can pass a parameter to test-contracts.sh. If you want to run everything, just don't pass any parameter.